bats/tests/common-setup.bash

```diff
@@ -35,6 +35,10 @@ _common_setup_file() {
     log DEBUG "Cloning Pyrsia repo: repo dir: ${PYRSIA_TEMP_DIR}, repo: ${GIT_REPO}, branch: ${GIT_BRANCH}"
     git clone --branch $GIT_BRANCH $GIT_REPO $PYRSIA_TEMP_DIR
   fi
+  local latest_commit;
+  latest_commit=$(git -C /tmp/pyrsia_tests/pyrsia log -1 --oneline);
+  log INFO "Integration tests are running using this version of Pyrsia: ${latest_commit}"
+  log INFO "For more details, refer to https://github.com/pyrsia/pyrsia/commit/$(echo "${latest_commit}" | awk '{print $1}') (This link works properly only when you test pyrsia/pyrsia repository.)"
```

Comment:
It is also possible to show the link to a PR, but I chose to just jump to a commit because the script would be complicated due to a command like `echo $latest_commit | sed -r 's/.*\(#([0-9]*)\)/\1/'`

Reply:
To be exact, of course this link does not work for other repo, so I added `(This link works properly only when you test pyrsia/pyrsia repository.)`... in case...